website/blog/2020-11-10-lithium.md

```diff
@@ -0,0 +1,502 @@
+using scalafix configuration file. This file can be either `.scalafix.conf` in
+the current workspace or an absolute file specified in
+`metals.scalafixConfigPath` user setting. It's important to note that the new
+code action is consistent with how sbt's scalafix plugin will behave.
+
+An example scalafix configuration for organize imports rule can look like this:
+
+```scala
+OrganizeImports {
+  groups = ["javax?\\.", "scala.", "*"]
+  removeUnused = true
+}
+```
+
+This will sort imports into 3 groups defined with regexes and remove any unused
+ones. Specifically, it will turn:
+
+```scala
+import scala.collection.mutable.{Buffer, ArrayBuffer}
+import java.time.Clock
+import java.lang.{Long => JLong, Double => JDouble}
+
+object RemoveUnused {
+  val buffer: ArrayBuffer[Int] = ArrayBuffer.empty[Int]
+  val long: JLong = JLong.parseLong("0")
+}
+```
+
+into
+
+```scala
+import java.lang.{Long => JLong}
+
+import scala.collection.mutable.ArrayBuffer
+
+object RemoveUnused {
+  val buffer: ArrayBuffer[Int] = ArrayBuffer.empty[Int]
+  val long: JLong = JLong.parseLong("0")
+}
+```
+
+More information can be found in the
+[liancheng/scalafix-organize-imports](https://github.com/liancheng/scalafix-organize-imports)
+repository.
+
+## Show implicits and type decorations
+
+Another highly anticipated feature was the ability to show additional
+information about the code, which is not provided explicitly. In this new
+release, users can use two new options when looking through their code:
+
+- `metals.showImplicitArguments` will enable users to see implicit parameters
+  within their code:
+
+![params](https://i.imgur.com/vAvM7YV.png)
+
+- `metals.showInferredType` will enable users to see inferred type for any
+  generic methods they are using:
+
+![types](https://i.imgur.com/K3QrUd2.png)
+
+Both new options are disabled by default, since they add a great number of
+information, which might not be necessary for all users. Full name with package
+for each of the additional types or values will be available on hover.
+
+Unfortunately, due to current limitations, additional decorations are only
+possible in Visual Studio Code. In other editors the additional information is
+available via hover and new `With synthetics added` section, which shows how the
+whole current line would look with the additional decorations.
+
+Example of how this alternative approach looks in Vim:
+
+![vim-sample](https://user-images.githubusercontent.com/13974112/94804295-e094a280-03ea-11eb-8282-2a783446e7c8.png)
+
+## Improved sbt BSP support
+
+In recent months, [eed3si9n](https://github.com/eed3si9n) and
+[adpi2](https://github.com/adpi2) worked, under the auspices of Scala Center, on
+making sbt capable of acting as a
+[Build Server Protocol](https://build-server-protocol.github.io/) server. This
+enabled Metals and other editors such as Intellij IDEA to directly communicate
+with sbt in order to compile the users' code.
```

Comment:
This is super subtle, but `user's` and `users'` have a slightly different meaning. `user's` would be correct here instead of `users'`. `users'` indicates possession on a collective users code, but `user's` signifies possession on a single users code.
```suggestion
with sbt in order to compile the user's code.
```

Reply:
I did mean collective users code, but you are right that might not work in this context.

Reply:
Fixed